snapcraft/extensions/kde_neon_6.py

```diff
@@ -278,7 +352,11 @@ def get_parts_snippet(self) -> Dict[str, Any]:
         # We can change this to the lightweight command-chain when
         # the content snap includes the desktop-launch from
         # https://github.com/canonical/snapcraft-desktop-integration
-        source = get_extensions_data_dir() / "desktop" / "kde-neon-6"
+        source = get_extensions_data_dir() / "desktop" / "command-chain-kde"
```

Comment:
```suggestion
        source = get_extensions_data_dir() / "desktop" / "command-chain"
```

Reply:
This file should be same for all. This file comes from here, but needs some fixes.

https://github.com/canonical/snapcraft/tree/main/extensions/desktop/command-chain

Reply:
How is 

set -- "${SNAP}/gnome-platform/command-chain/desktop-launch" "$@"
# shellcheck source=/dev/null
source "${SNAP}/snap/command-chain/run"

Going to help me?????? Please don't do this now.